README.md

```diff
@@ -221,7 +221,7 @@ See also [Network checker](docs/netcheck.md).
 
 - [nginx](https://kubernetes.github.io/ingress-nginx): the NGINX Ingress Controller.
 
-- [metallb](docs/metallb.md): the MetalLB bare-metal service LoadBalancer provider.
+- [metallb](docs/ingress_controller/metallb.md): the MetalLB bare-metal service LoadBalancer provider.
```

Comment:
That is a nice catch, but I think metallb.md should be moved to docs/ from docs/ingress_controller instead, because MetalLB is one of LoadBalancer services, not ingresses.

Reply:
Yes that was confusing because it is not an ingress controller. Okay I moved it.